azkaban-exec-server/src/main/java/azkaban/execapp/action/KillJobAction.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.execapp.action;
+
+import azkaban.ServiceProvider;
```

Comment:
order is already optimized by https://github.com/dubreuia/intellij-plugin-save-actions. We found it very helpful. Maybe have a try and adopt it in your team if you feel the same way.